supply-chain/platform-engineering/how-to/installing-supply-chain/post-install-configuration.hbs.md

```diff
@@ -4,80 +4,86 @@
 
 This topic tells you what to configure to complete your Tanzu Supply Chain installation.
 
-After you have installed Tanzu Supply Chain, there are several steps you must complete to ensure you have a functioning installation.
+After you have installed Tanzu Supply Chain, there are several steps you must complete to ensure
+you have a functioning installation. Use Namespace Provisioner to configure service accounts and permissions.
 
-## Namespace Provisioner
+## Things to configure
 
-The recommended way of configuring service accounts and permissions on Tanzu Application Platform is
-to use the Namespace Provisioner.
+- OCI Store Configuration
+  - Supply Chains persist data between stages by reading and writing to an OCI repository.  The location of the OCI repository is configured by a Kubernetes Secret named `oci-store` that exists within the developer namespace.
+  - Access to this repository is controlled by a tekton annotated secret which can have any name as long as it has the `tekton.dev/docker-0` annotation pointing to the OCI repository.
+- Permissions for Buildpacks Cluster Builders for buildpack-build component
+  - If you are planning to use the `buildpack-build` component to create images using TBS configured with `ClusterBuilders` you must add some additional permissions.
 
-```shell
-❯ kubectl create namespace my-supply-chains
-❯ kubectl label namespaces my-supply-chains apps.tanzu.vmware.com/tap-ns=""
+## Setup using Namespace Provisioner
+
+Recommended way is to use the Namespace provisioner for the entire setup:
+
+**Step 1**: is to create a `Secret` in the `tap-install` namespace that has the location and credentials for the `oci-store` as follows:
+
+```console
+cat << EOF | kubectl apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: supply-chain-oci-store-credentials
+  namespace: tap-install
+type: Opaque
+stringData:
+  ocistore.yaml: |
+    tanzusupplychain:
+      ocistore:
+        username: <registry-username>
+        password: <registry-password>
+        server: <registry-server>
+        repository: <registry-repo>
```

Comment:
Can you call out the expected value for these variables, and I will add them in.